google/cloud/bigtable/mutation_batcher_test.cc

```diff
@@ -167,12 +167,12 @@ class MutationBatcherTest : public bigtable::testing::TableTestFixture {
       EXPECT_CALL(*reader, StartCall).Times(1);
 
       EXPECT_CALL(*client_, PrepareAsyncMutateRows)
-          .WillOnce([reader, exchange](grpc::ClientContext* context,
-                                       btproto::MutateRowsRequest const& r,
-                                       grpc::CompletionQueue*) {
+          .WillOnce([this, reader, exchange](
+                        grpc::ClientContext* context,
+                        btproto::MutateRowsRequest const& r,
+                        grpc::CompletionQueue*) {
             EXPECT_STATUS_OK(IsContextMDValid(
```

Comment:
there is only one call. Maybe we just define `ValidateMetadataFixture fixture;` in here?

(I am saying this just for consistency. I do not care if you actually make this change.)

Reply:
Ditto, the surrounding function is called many times.

I checked again, and the places where I use a local fixture are (lucky for me) only called once.